reducers/gotPlayer/player.js

```diff
@@ -28,3 +31,19 @@ export default (state = initialState, action) => {
       return state;
   }
 };
+
+export const player = {
```

Comment:
I don't like how we basically have to redeclare all of the props here.

If we add a prop in the API, it's a lot of code to add just to get it to show up in the UI.

With Jade we could start editing the template immediately and just reference the new prop.